src/app/containers/Image/index.test.jsx

```diff
@@ -78,6 +78,11 @@ describe('Image', () => {
       <ImageContainer {...data} />,
     );
 
+    shouldShallowMatchSnapshot(
+      'should render a lazyload container instead of an image',
```

Comment:
```suggestion
      'should render a lazyload container instead of an image if the image as after the 3rd block',
```

We should be explicit in the test description